build-files/build-image.sh-ppc64le

```diff
@@ -64,4 +73,19 @@ run() {
   install_runner
 }
 
+ARCH=`uname -m`
+RUNNERREPO="https://github.com/actions/runner"
+SDK=7.0.100
+while getopts "a:s:" opt
+do
+    case ${opt} in
+        a)
+            RUNNERREPO=${OPTARG}
+            ;;
+        s)
+            SDK=${OPTARG}
+            ;;
+    esac
+done
+shift $(( OPTIND - 1 ))
```

Comment:
Again for Anup - see how simple this is?  We haven't done a lot of CLI style development, but we have many things in the work that should be aware of how to do this in a standard, simple way and this is a great practice while being extremely simple.  (I remember hand coding this logic in C and sh in decades long gone, oh how I love how far we've come ;-) )